### DISCUSSION TITLE
Phase 6 - Formalized

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#542] [Announcements - Changes Made]
## Tags
* `<podcast:txt>` by @tedhosmann - #543
* `<podcast:remoteItem>` by @agates - #481 
* `<podcast:podroll>` by @snookfin - #451 
* `<podcast:updateFrequency>` by @nathangathright - #515 
* `<podcast:podping>` by @brianoflondon and @agates - #517 
* `<podcast:valueTimeSplit>` by @agates - #512 

## Other Changes
* Phase 6 - Formalize by @daveajones in https://github.com/Podcastindex-org/podcast-namespace/pull/526
* add simplecast to txt element support by @sophia-chapman in https://github.com/Podcastindex-org/podcast-namespace/pull/530
* bLIP10: Change reply_custom_key to int by @Alwin-Stockinger in https://github.com/Podcastindex-org/podcast-namespace/pull/535
* Adding updateFrequency by @nathangathright in https://github.com/Podcastindex-org/podcast-namespace/pull/536
* Catch up to main by @daveajones in https://github.com/Podcastindex-org/podcast-namespace/pull/540
* Fix updateFrequency markdown by @nathangathright in https://github.com/Podcastindex-org/podcast-namespace/pull/541

## New Contributors
* @sophia-chapman made their first contribution in https://github.com/Podcastindex-org/podcast-namespace/pull/530

**Full Changelog**: https://github.com/Podcastindex-org/podcast-namespace/compare/v5.9...v6.0

**Comment by daveajones** [comment]
Thank you to ALL contributors.  This is a very technical phase with a lot of in the weeds stuff, but it'll prove to be the building blocks of awesome stuff coming next.

Beers on me.

🎉 🍻 ❤️

**Reply by snookfin** [reply]
When you get a chance can you link to the discussion/issue for each. I'm having trouble finding them all. 

Thanks @daveajones!

**Reply by daveajones** [reply]
Sure!  Edit: Done!

**Comment by daveajones** [comment]
XMLNS docs on the main website have also been updated:

https://podcastindex.org/namespace/1.0#txt
